src/plugins/intel_cpu/src/emitters/plugin/riscv64/jit_eltwise_emitters.hpp

```diff
@@ -261,5 +261,23 @@ class jit_subtract_emitter : public jit_emitter {
     void emit_isa(const std::vector<size_t>& in_vec_idxs, const std::vector<size_t>& out_vec_idxs) const;
 };
 
+class jit_xor_emitter : public jit_emitter {
```

Comment:
Please align the emitter name with op name `LogicalXor` - it should be `jit_logical_xor_emitter`.

Ans also please align your changes with others in alphabetical order 😊